schemas/layer-2.cue

```diff
@@ -1,84 +1,131 @@
 package schemas
 @go(layer2)
 
+// Layer2 describes a collection or catalog of technology-specific, threat-informed security controls
+// that can be applied to an information system.
 #Layer2: {
+    // metadata about the Layer2 collection
     metadata?: #Metadata
-
+    // one or more ControlFamily objects
     "control-families"?: [...#ControlFamily] @go(ControlFamilies)
+    // zero or more Threats that are mitigated by the controls in the ControlFamilies
     threats?: [...#Threat] @go(Threats)
+    // zero or more Capabilities in use that inform the controls in the ControlFamilies
     capabilities?: [...#Capability] @go(Capabilities)
-
+    // zero or more mapping references to controls defined in this or other Layer2 frameworks or standards
     "shared-controls"?: [...#Mapping] @go(SharedControls)
+    // zero or more mapping references to threats defined in this or other Layer2 frameworks or standards
     "shared-threats"?: [...#Mapping] @go(SharedThreats)
+    // zero or more mapping references to capabilities defined in this or other Layer2 frameworks or standards
     "shared-capabilities"?: [...#Mapping] @go(SharedCapabilities)
 }
 
-// Resuable types //
-
 #Metadata: {
+    // unique identifier for the Layer2 collection
     id: string
+    // name for the Layer2 collection
     title: string
+    // description of the Layer2 collection
     description: string
     version?: string
+    // timestamp of the last modification to the Layer2 collection
     "last-modified"?: string @go(LastModified)
+    // Applicability categories are facets that apply to the controls in this Layer2 collection. These facet values can be used to filter which controls from Layer2 are used in Layer 3 or Layer 4.
     "applicability-categories"?: [...#Category] @go(ApplicabilityCategories)
+    // List of applicable references to Layer 1 guidance, technical capabilities and threats that inform the Layer2 collection
     "mapping-references"?: [...#MappingReference] @go(MappingReferences)
```

Comment:
See https://github.com/ossf/security-baseline/blob/main/baseline/frameworks.yaml for an example

Reply:
++ This should also allow Baseline to avoid putting guidance links in the Lexicon as those could be auto-generated from this data